packages/rainbowkit/src/components/RainbowKitProvider/AuthenticationContext.tsx

```diff
@@ -8,6 +8,11 @@ import React, {
 } from 'react';
 import { useAccount } from 'wagmi';
 
+interface ConnectorData {
+  account: string;
+  chain: number;
+}
+
```

Comment:
Can we infer this type import from `@wagmi/core` directly?

Reply:
Added their `ConnectorData` type. Here is the type for more info: [wagmi type](https://github.com/wagmi-dev/wagmi/blob/38306606d2fd72a4c6918323bf86a1afda348638/packages/connectors/src/base.ts#L14)